pkg/errors/error.go

```diff
@@ -41,4 +41,5 @@ var (
 	ErrClusterNotFound         = errors.New("Cluster instance not found in store")
 	ErrDuplicateBrickPath      = errors.New("Duplicate brick entry")
 	ErrVolFileNotFound         = errors.New("volume file not found")
+	ErrInvalidQuery            = errors.New("Invalid query paramater")
```

Comment:
Remove this. Return a specific error as commented above.

Reply:
done